operators.js

```diff
@@ -387,6 +387,10 @@ function descending() {
   return (ops) => updateMeta(ops, 'descending', true)
 }
 
+function sortBy(sortOption) {
+  return (ops) => updateMeta(ops, 'sortBy', sortOption)
+}
+
```

Comment:
Instead of this, could we have the operators `sortByArrival` and `sortByCreation`? It's easy to make mistakes with string APIs, and those mistakes would be totally silent (no error thrown). This would make it similar to `descending()`, note that we don't have the operator `order('descending')` and `order('ascending')`, we only have `descending()`. In that light, we can also drop `sortByCreation()` and have only `sortByArrival()`, since the default is creation.